Applications/Spire/Include/Spire/Ui/StaticDropDownMenu.hpp

```diff
@@ -41,6 +41,8 @@ namespace Spire {
       StaticDropDownMenu(std::vector<QVariant> items,
         const QString& display_text, QWidget* parent = nullptr);
 
+      virtual ~StaticDropDownMenu() = default;
```

Comment:
Why was this added?

Reply:
Right, forgot about the base class' virtual destructor.